tests/conftest.py

```diff
@@ -29,18 +116,31 @@ def app():
 
     try:
         clear_database()
-    except Exception:
+    except Exception as e:
         pass
 
 
 @pytest.fixture
 def user(app):
-    app.auth.add("test", "test")
+    engine = create_engine(POSTGRES_CONNECTION)
+    driver = SQLAlchemyAuthDriver(POSTGRES_CONNECTION)
+
+    try:
+        driver.add("test", "test")
+    except Exception as e:
+        pass
+
     yield {
         "Authorization": ("Basic " + base64.b64encode(b"test:test").decode("ascii")),
         "Content-Type": "application/json",
     }
-    app.auth.delete("test")
+
+    try:
+        driver.delete("test")
```

Comment:
the `add` and `delete` functions still exist in the auth driver, why does this need to change?

Reply:
This ones a little complicated. Its the way tests are set. I did it a while ago and idk why it doesnt work but when i remove the driver, the tests start to fail